pkg/cmd/internal/issues/issues.go

```diff
@@ -385,6 +387,8 @@ func (p *poster) post(origCtx context.Context, formatter IssueFormatter, req Pos
 			ctx, p.Org, p.Repo, *foundIssue, &comment); err != nil {
 			return errors.Wrapf(err, "failed to update issue #%d with %s",
 				*foundIssue, github.Stringify(comment))
+		} else {
+			p.l.Printf("created comment on existing GitHub issue")
```

Comment:
Could we also log the issue # for easier cross-referencing?

Reply:
Done.